addons/stock_account/models/product.py

```diff
@@ -460,6 +462,7 @@ def _svl_empty_stock_am(self, stock_valuation_layers):
         move_vals_list = []
         product_accounts = {product.id: product.product_tmpl_id.get_product_accounts() for product in stock_valuation_layers.mapped('product_id')}
         for out_stock_valuation_layer in stock_valuation_layers:
+            out_stock_valuation_layer = out_stock_valuation_layer.with_company(out_stock_valuation_layer.company_id)
```

Comment:
_account_entry_move does the force_company/with_company so i'm not sure it's useful here?